src/components/anti-everyone.ts

```diff
@@ -34,6 +39,18 @@ export default class AntiEveryone extends BotComponent {
             await message.reply({
                 content: `Did you really just try to ping ${memberCount} people?`,
             });
+            if(!replies.has(message.author.id) replies[message.author.id] = [];
```

Comment:
It looks like there's a missing `)` here

Reply:
Yeah, I was originally coding this on directly on GitHub, so I missed that )